subprojects/docs/src/docs/userguide/dependency_locking.adoc

```diff
@@ -29,6 +29,9 @@ Now dependencies can be declared against major or minor version range, enabling
 * Teams that want to always use the latest of their dependencies can use dynamic versions, locking their dependencies only for releases.
 The release tag will contain the lock states, allowing that build to be fully reproducible when bug fixes need to be developed.
 
+Combined with <<publishing_maven.adoc#publishing_maven:resolved_dependencies,publishing resolved versions>>, you can also hide the declared dynamic version part at publication.
+And consumers will instead see the versions that your release resolved.
```

Comment:
It reads strange to have a sentence starting with "And"

Reply:
Fixed